playbooks/common/README.md

```diff
@@ -0,0 +1,8 @@
+# Common playbooks
+
+This directory has a generic set of playbooks that run against different
+"providers" through use of the [`bin/cluster`](../../bin/cluster) tool. The
+tool is now deprecated.
+
+Playbooks in this directory use generic group names that do not line up with the
+groups used by the `byo` playbooks or `bin/cluster` derived playbooks.
```

Comment:
Needs to specify that these playbooks back the byo playbooks as well as the other "provider" playbooks.